invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v2.0.0.json

```diff
@@ -363,12 +368,17 @@
               }
             },
             "resource_type" : {
-              "properties" : {
-                "subtype" : {
-                  "type" : "keyword"
+              "type": "object",
+              "properties": {
+                "@v": {
+                  "type": "keyword"
                 },
-                "type" : {
-                  "type" : "keyword"
+                "id": {
+                  "type": "keyword"
+                },
+                "title": {
+                  "type": "object",
+                  "dynamic": true
```

Comment:
Since this is the old mapping, we should leave this file alone and not provide these changes. Previous resource types should operate the same as before.

Reply:
true, nice catch sorry about that